bin/build

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+TAG=latest
+
+while getopts "t:d:r:nh" opt; do
```

Comment:
Can you add options for REF and ORG?

For example, if REF is specified, use that for MIQ_REF, SUI_REF and APPLIANCE_REF in manageiq-base image.

Reply:
Hm, I'm not sure I'd want to always use the same ref for all of them. When developing it's much more likely that I'll need to change the miq ref, but won't want the same value to apply to the appliance or sui ref. Maybe I'll push those six values into env vars rather than options.

Reply:
I'll take anything as long as REF can be changed 😄